### PR TITLE
Updated CppUTest to 3.7.1

### PR DESCRIPTION
**Pull request by basvodde**
Updated the CppUTest version from 3.6 to 3.7.1.

### CODE REVIEW

File: Library/Formula/cpputest.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class Cpputest < Formula
   homepage "http://www.cpputest.org/"
-  url "https://github.com/cpputest/cpputest/archive/v3.6.tar.gz"
-  sha1 "308a4200adfb86182251d435e09f42360d9ed8ea"
+  url "https://github.com/cpputest/cpputest.github.io/blob/master/releases/cpputest-3.7.1.tar.gz?raw=true"
```
Comment (xu-cheng) [inline Library/Formula/cpputest.rb]: `"https://github.com/cpputest/cpputest/archive/v3.7.1.tar.gz"`

**Comment by basvodde** [comment]
Ok, changed the url to exclude the raw=true.

**Comment by jacknagel** [comment]
Change it to exactly "https://github.com/cpputest/cpputest/archive/v3.7.1.tar.gz"

**Comment by basvodde** [comment]
Changed and bumped up the version one minor-minor one to include a build failure on exotic platforms.

**Comment by mikemcquaid** [comment]
Did this build for you locally? There's no `./configure`: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/26282/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_cpputest/

**Comment by basvodde** [comment]
I installed the formula locally, let me check the build...

**Comment by basvodde** [comment]
Ok, obviously I was asleep while testing locally. Sorry about that.

As requested above, I had changed the link to 
https://github.com/cpputest/cpputest/archive/v3.7.2.tar.gz

While I was wondering why. Now I know why this is incorrect. The tgz to that link contains the source, but it doesn't contain the distribution release. That one is at a separate link. So, I'll update this once more back to the original link:
https://github.com/cpputest/cpputest/releases/download/3.7.2/cpputest-3.7.2.tar.gz

This is the packaged version. As the configure file is generated and not version controlled, it is in the distribution release but not in the source version (which github has made automatically)

Updating it now.

**Comment by basvodde** [comment]
Ok, done now. Sorry for the confusion, now it downloads the right tgz

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by mikemcquaid** [comment]
For future reference the preferred commit message format for simple version updates is `cpputest 3.7.2`.

**Pull request closed without merging**